cmd/podman/varlink/io.podman.varlink

```diff
@@ -393,34 +372,29 @@ type Runlabel(
     opts: [string]string
 )
 
-# Ping provides a response for developers to ensure their varlink setup is working.
-# #### Example
-# ~~~
-# $ varlink call -m unix:/run/podman/io.podman/io.podman.Ping
-# {
-#   "ping": {
-#     "message": "OK"
-#   }
-# }
-# ~~~
-method Ping() -> (ping: StringResponse)
-
-# GetVersion returns a Version structure describing the libpod setup on their
-# system.
-method GetVersion() -> (version: Version)
+# Returns version information
+method GetVersion() -> (
+    version: string,
+    go_version: string,
+    git_commit: string,
+    built: string, # as RFC3339
+    os_arch: string,
+    remote_api_version: int
+)
 
 # GetInfo returns a [PodmanInfo](#PodmanInfo) struct that describes podman and its host such as storage stats,
 # build information of Podman, and system-wide registries.
 method GetInfo() -> (info: PodmanInfo)
 
-# ListContainers returns a list of containers in no particular order.  There are
-# returned as an array of ListContainerData structs.  See also [GetContainer](#GetContainer).
-method ListContainers() -> (containers: []ListContainerData)
+# ListContainers returns information about all containers.
+# See also [GetContainer](#GetContainer).
+method ListContainers() -> (containers: []Container)
 
-# GetContainer takes a name or ID of a container and returns single ListContainerData
-# structure.  A [ContainerNotFound](#ContainerNotFound) error will be returned if the container cannot be found.
-# See also [ListContainers](ListContainers) and [InspectContainer](#InspectContainer).
-method GetContainer(name: string) -> (container: ListContainerData)
+# Returns information about a single container.  If a container with the given
```

Comment:
I think we'd prefer that all descriptions start with the name of the API endpoint, for consistency (so `GetContainer returns...` instead)

Reply:
Ooh I hadn't noticed that, but I like it. Changed the ones I touched to be consistent again.